repair/row_level.cc

```diff
@@ -47,6 +47,7 @@
 #include "gms/gossiper.hh"
 #include "repair/row_level.hh"
 #include "mutation_source_metadata.hh"
+#include <absl/container/flat_hash_set.h>
```

Comment:
Looks unrelated.

Reply:
Fixed. It was an leftover when playing with flat_hash_set.